src/main/java/picard/illumina/ClusterDataToSamConverter.java

```diff
@@ -42,26 +43,28 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.StringJoiner;
+
+import static picard.util.IlluminaUtil.BARCODE_DELIMITER;
```

Comment:
I think we should avoid static imports, since they make the code harder to read without using an IDE, with an exception for test libraries such as testng's `Assert` or hamcrest style matchers in test code.

Reply:
oops.